### PR TITLE
refine: thread safety write intro data

### PR DESCRIPTION
**Pull request by shhdgit**

### CODE REVIEW

**Review by baurine — COMMENTED** [review]
lgtm

File: pkg/utils/distro/distro.go [hunk 1]
```diff
@@ -14,12 +14,26 @@
...
+var data atomic.Value
+
+func Replace(distro introData) {
+	data.Store(distro)
 }
 
 func Data(k string) string {
-	return data[k].(string)
+	d := data.Load().(introData)
+	if d[k] == nil {
```
Comment (breeswish) [inline pkg/utils/distro/distro.go:34]: When will this happen?

File: pkg/utils/distro/distro.go [hunk 2]
```diff
@@ -14,12 +14,26 @@
...
+		log.Info("Distro information doesn't exist", zap.String("key", k))
```
Comment (breeswish) [inline pkg/utils/distro/distro.go:35]: Maybe we can remove this log! Since we can just know the missing key from the output.

Comment (shhdgit) [reply pkg/utils/distro/distro.go]: Yeah, let's go!

**Review by breeswish — APPROVED** [review]

**Pull request merged**